python/lsst/cp/pipe/defects.py

```diff
@@ -277,13 +291,18 @@ def runDataRef(self, dataRef, visitList):
 
         defectLists = {'dark': [], 'flat': []}
 
+        midTime = 0
+        filters = set()
+
         if self.config.mode == 'MASTER':
             if len(visitList) > 1:
                 raise RuntimeError(f"Must only specify one visit when using mode MASTER, got {visitList}")
-            dataRef.dataId['visit'] = visitList[0]
+            dataRef.dataId['expId'] = visitList[0]
```

Comment:
Should visitList become exposureList, to reduce possible confusion?

Reply:
Yeah, and the rest! There are currently 75 other instances of the string `visit` in that file alone. I was concentrating on the code for now, because I was in a hurry, but definitely do agree that ideally all instances would be changed...

How about we ticket doing that once Andrés' addition of tests for defects merge, so that 1) we have better test coverage when renaming variables, and 2) I don't give him a rebase nightmare to deal with by changing 75 instances on him?